BREAKING-CHANGES.md

```diff
@@ -38,7 +39,32 @@ This is a quick summary of the sections below:
 
 ## Unreleased (v0.24.1)
 
-### `Table::widths()` now accepts `AsRef<[Constraint]>` ([#628])
+### `Table::new()` now requires specifying the widths of the columrs (#664)
+
+
+Previously `Table`s could be constructed without widths. In almost all cases this is an error.
```

Comment:
> In almost all cases this is an error.

I'm curious, did we get any bug reports about this + what kind of an errors users get if no widths are specified?

Reply:
Great, that makes sense.